docs/content/Coding Milestones/PoC2023q1/example1.md

```diff
@@ -33,12 +33,24 @@ they appear in this example.
    end="<!--example1-pre-kcp-end-->"
 %}
 
+{%
+   include-markdown "example1-subs/example1-space-manager.md"
```

Comment:
example1-pace-manager-start ?

Reply:
If we do this, the next line will look like: `start="<!--example1-space-manager-start-start-->"`